### PR TITLE
Added Screen: Settings

### PR DESCRIPTION
**Pull request by phil1995**
The settings screen was created as specified in #9, with the exception that iCloud Drive is not listed under "Cloud Services".
This is because we cannot log the user out of their Apple ID directly from the app, moreover it is questionable whether the user would be aware of the implications of this decision.

Debug mode is automatically enabled for debug builds, so to test disabling debug mode, the preprocessor macro block in `CryptomatorUserDefaults` must be commented out.

### CODE REVIEW

File: CryptomatorCommon/Sources/CryptomatorCommonCore/CryptomatorUserDefaults.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	public var debugModeEnabled: Bool {
+		get {
+			#if DEBUG
+			true
+			#else
+			read() ?? false
+			#endif
+		}
+		set { write(value: newValue) }
+	}
```
Comment (tobihagemann) [inline CryptomatorCommon/Sources/CryptomatorCommonCore/CryptomatorUserDefaults.swift:41]: A different approach could be to have a different default value for the debug environment. Something like:

```swift
#if DEBUG
private static let debugModeEnabledDefaultValue = true
#else
private static let debugModeEnabledDefaultValue = false
#endif

public var debugModeEnabled: Bool {
	get { read() ?? CryptomatorUserDefaults.debugModeEnabledDefaultValue }
	set { write(value: newValue) }
}
```

File: Cryptomator/Common/CloudAccountList/AccountListViewController.swift [hunk 2]
```diff
@@ -127,6 +127,7 @@ class AccountListViewController: SingleSectionTableViewController {
 	}
 
 	override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+		tableView.deselectRow(at: indexPath, animated: true)
```
Comment (tobihagemann) [inline Cryptomator/Common/CloudAccountList/AccountListViewController.swift:130]: Not sure if I'm nitpicky here. Would it somehow be possible to "know" if a deselect is even necessary? Table view cells have this "magic" behavior with a deselect animation when using the swipe gesture to go back a view. This gets lost if the row gets deselected prematurely.

Comment (phil1995) [reply Cryptomator/Common/CloudAccountList/AccountListViewController.swift:130]: I think we should postpone this until we start our general refactoring of the `UITableViewControllers`.
But you're right, we should definitely keep this in mind.

File: CryptomatorCommon/Sources/CryptomatorCommonCore/CryptomatorUserDefaults.swift (hunk 1 above)
Comment (phil1995) [reply CryptomatorCommon/Sources/CryptomatorCommonCore/CryptomatorUserDefaults.swift]: Yes, this is indeed a better approach, as it also allows to manually turn off the debug mode for debug builds.
I have updated that part of the code, thank you.

**Pull request merged**